packages/webapp/src/delegates/api/eden-contract.ts

```diff
@@ -18,14 +17,57 @@ const getMemberBudgetBalance = () => {
     return {}; // TODO
 };
 
-const getMemberElectionParticipationStatus = () => {
-    return {}; // TODO
-};
+const MEMBER_REPRESENTATIVE_IF_NOT_PARTICIPATED_IN_RECENT_ELECTION =
+    "zzzzzzzzzzzzj";
+// check that member has participated in an election (if there's been one yet) (!="zzz...") and came to consensus with their group in the last election (!=0)
+const memberHasRepresentative = (member: EdenMember) =>
+    member.account !== "" &&
+    member.account !==
+        MEMBER_REPRESENTATIVE_IF_NOT_PARTICIPATED_IN_RECENT_ELECTION;
```

Comment:
`const memberHasRepresentative = (member: EdenMember) => member?.account !== MEMBER_REPRESENTATIVE_IF_NOT_PARTICIPATED_IN_RECENT_ELECTION;` ?

Reply:
2 distinct scenarios. I've added another constant to document the difference